src/main/java/org/traccar/notification/NotificationFormatter.java

```diff
@@ -63,7 +63,7 @@ public static FullMessage formatFullMessage(long userId, Event event, Position p
         return TextTemplateFormatter.formatFullMessage(velocityContext, event.getType());
     }
 
-    public static String formatShortMessage(long userId, Event event, Position position) {
+    public static ShortMessage formatShortMessage(long userId, Event event, Position position) {
```

Comment:
Same here. We probably don't need a separate method.

Reply:
This has been merged into "FormatMessage" in 9fd24af.